File: Projekt_Python/zwierzeta/Owca.py
from Zwierze import Zwierze

class Owca(Zwierze): 
    def __init__(self, x, y,dane):
        if dane==None:
            self._x=x
            self._y=y
            self._xprev=x
            self._yprev=y
            self._symbol='O'
            self._sila=4
            self._inicjatywa=4
            self._zycie=0
            self._wykonal_ruch=True
            self._numer=1
        else:
            self._x = dane[0]
            self._y = dane[1]
            self._xprev = dane[2]
            self._yprev = dane[3]
            self._inicjatywa = dane[4]
            self._symbol = 'O'
            self._sila = dane[5]
            self._wykonal_ruch = False
            self._zycie = dane[6]
            self._numer = dane[7]
        self._kolor= (224,255,255)

File: projekt_java/src/po/java/zwierzeta/Owca.java
package po.java.zwierzeta;

import po.java.Zwierze;

import java.awt.*;

public class Owca extends Zwierze {
    public Owca(int x,int y) {
        this.x=x;
        this.y=y;
        this.xprev=x;
        this.yprev=y;
        this.symbol='O';
        this.sila=4;
        this.inicjatywa=4;
        this.zycie=0;
        this.wykonal_ruch=true;
        this.numer=1;
        this.kolor= Color.CYAN;
    }
    public Owca(int []dane)
    {
        this.x = dane[0];
        this.y = dane[1];
        this.xprev = dane[2];
        this.yprev = dane[3];
        this.inicjatywa=dane[4];
        this.symbol = 'O';
        this.sila = dane[5];
        this.wykonal_ruch = false;
        this.zycie = dane[6];
        this.numer=dane[7];
        this.kolor= Color.CYAN;
    }
}
